### PR TITLE
EE-1157: Transfer to account support in AS.

### PR DESCRIPTION
**Pull request by mpapierski**
Ref: https://casperlabs.atlassian.net/browse/EE-1157
Ref: https://casperlabs.atlassian.net/browse/EE-1113

### CODE REVIEW

File: smart_contracts/contract_as/assembly/purse.ts [hunk 1]
```diff
@@ -200,3 +219,53 @@ export function transferFromPurseToPurse(sourcePurse: URef, targetPurse: URef, a
...
+
+/**
+ * Transfers `amount` of motes from main purse purse to `target` account.
+ * If `target` does not exist it will be created.
+ *
+ * @param amount Amount is denominated in motes
+ * @returns This function will return a [[TransferredTo.TransferError]] in
+ * case of transfer error, in case of any other variant the transfer itself
+ * can be considered successful.
+ * @hidden
```
Comment (henrytill) [inline smart_contracts/contract_as/assembly/purse.ts:231]: Should this public?

**Review by henrytill — APPROVED** [review]
LGTM, but perhaps `transferToAccount` should be public?

File: smart_contracts/contract_as/assembly/purse.ts (hunk 1 above)
Comment (mpapierski) [reply smart_contracts/contract_as/assembly/purse.ts]: Done

**Pull request merged**